app/models/rbac.rb

```diff
@@ -121,20 +120,13 @@ def self.get_self_service_objects(user, miq_group, klass)
     klass.user_or_group_owned(user, miq_group).select(minimum_columns_for(klass))
```

Comment:
This `select` is useless since 9a388360a4e76a97284a1dc025ee316bdc560b1f switched to calling pluck on the result.

Can/should we put the `.except(:order)` here?